src/docs/workflows/device-commissioning.mdx

```diff
@@ -0,0 +1,116 @@
+
+### Number of Devices Commissioned at Once
+
+Do you expect a residential user to unbox 16 devices at once? Maybe after 2 or 3 devices, they already remembered all the steps. Your application can remember their preferences so that they don't have to redo things repeatedly.
+
+Or maybe a technician is trying to set up an electricity management system for their client, and they try importing more than a thousand devices at a time? If the technician uses a batch configuration, should this process be run in the background while the technician is configuring something else?
+
+### Expertise of Users
+
+The design should be tuned to fit the need of different personas. They can be non-technical users, operations / analysts, your client's engineers, your own company's engineers, or site managers. Consumer applications would demand more instructions (in texts, images, animations, or even AR) and more breaking down in the steps. At the same time, technicians generally desire more efficiency in the setup process.
+
+### Platform
+
+If a user is doing the device commissioning from mobile phones and tablets, they will have the advantage of a phone camera which can be helpful for scanning QR codes. Other technologies like "BlinkUp" allow device sensors to receive data via your phone's screen blinking. Mobile phones are easier to carry around and can make your physical location less messy, especially if users have instruction manuals and wires flying around everywhere.
+
+On the other hand, computers are better for precise configuration and can process many device details simultaneously due to their superior computing power. The screen size is larger, meaning that you can fit more information and interactions in one screen.
+
+# Design Tricks
+
+## Do not force your users to make decisions on the first go.
+
+If your user does not specify where a device is physically located, will that stop your dashboard from working? Or is it safe to list this device as "locations unspecified" for a while, let the user know that they can skip these settings for now and come back in the future?
+
+## The technician who set it up may not be the final user.
+
+Often, users would invite technicians to come to their home or company and set up the system for them, and later on, the technician would pass the admin right to the user. Sometimes this necessitates a "handoff" experience at the end of your workflow.
+
+## Do not force them to create an account until the need comes up.
+
+Accounts are often used to control the device from anywhere (web, phone, device's HMI) or manage access for different people. However, not all of these functionalities are necessary during an initial device setup. You should only prompt your users to create the account when they actually need it, rather than making the onboarding process burdensome.
+
```

Comment:
```suggestion
## Do not force them to create an account until needed

Accounts are often used to control the device from anywhere (web, phone, device's HMI) or manage access for different people. However, not all of these functionalities are necessary during an initial device setup. You should only prompt your users to create the account when they actually need it, rather than adding yet another step to the commissioning process.

```